core/arch/arm/kernel/link.mk

```diff
@@ -217,15 +217,12 @@ $(link-out-dir)/tee.bin: $(link-out-dir)/tee-pager.bin \
 			 $(link-out-dir)/tee-init_mem_usage.txt \
 			./scripts/gen_hashed_bin.py
 	@$(cmd-echo-silent) '  GEN     $@'
-	$(q)./scripts/gen_hashed_bin.py \
+	$(q)load_addr=$(shell cat $(link-out-dir)/tee-init_load_addr.txt); \
```

Comment:
Shouldn't that `;`  be replaced by `&&` to error out early in case the file can't be read?